helm_deploy/values-dev.yaml

```diff
@@ -20,7 +20,12 @@ generic-service:
     BANNER_DPSMENU:
 
   allowlist:
-    hpa-iis-stage: "51.104.41.208/32"
+    hpa-iis-stage-1: "51.141.12.82/32"
+    hpa-iis-stage-2: "51.141.12.83/32"
+    hpa-iis-stage-3: "51.141.12.84/32"
+    hpa-iis-stage-4: "51.141.12.85/32"
```

Comment:
was expecting them to be different values?

Reply:
yes, thanks for the question...updated to correct staging outbound address